UI tests failing

Last UI test success was on May 24
https://github.com/mozilla-mobile/android-components/commit/bb1db4162304da94e2364fcb5fa34e80f092a8c0
There were several PRs merged with no taskcluster run then afterwards tests began to fail:
https://github.com/mozilla-mobile/android-components/commit/bb1db4162304da94e2364fcb5fa34e80f092a8c0

logcat:
```
 Task :support-android-test:compileDebugJavaWithJavac
> Task :support-android-test:processDebugJavaRes NO-SOURCE
> Task :support-android-test:transformClassesAndResourcesWithPrepareIntermediateJarsForDebug
> Task :feature-session-bundling:kaptGenerateStubsDebugAndroidTestKotlin
> Task :feature-session-bundling:kaptDebugAndroidTestKotlin

> Task :feature-session-bundling:compileDebugAndroidTestKotlin FAILED
e: warnings found and -Werror specified
w: /build/android-components/components/feature/session-bundling/src/androidTest/java/mozilla/components/feature/session/bundling/OnDeviceSessionBundleStorageTest.kt: (41, 35): 'SessionBundleStorage' is deprecated. This component is getting removed. See feature-tab-collections for an alternative.
w: /build/android-components/components/feature/session-bundling/src/androidTest/java/mozilla/components/feature/session/bundling/OnDeviceSessionBundleStorageTest.kt: (57, 19): 'SessionBundleStorage' is deprecated. This component is getting removed. See feature-tab-collections for an alternative.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':feature-session-bundling:compileDebugAndroidTestKotlin'.
> Compilation error. See log for more details
```

--- a/automation/taskcluster/decision_task.py
+++ b/automation/taskcluster/decision_task.py
@@ -142,7 +142,7 @@ def create_module_tasks(module):
     elif "lintTask" in override:
         gradle_tasks = " ".join([
             gradle_module_task_name(module, gradle_task)
-            for gradle_task in ['assemble', 'test', override["lintTask"]]
+            for gradle_task in ['assemble', 'assembleAndroidTest', 'test', override["lintTask"]]
         ])
         task_definitions.append(
             BUILDER.craft_build_task(
@@ -158,7 +158,7 @@ def create_module_tasks(module):
     else:
         gradle_tasks = " ".join([
             gradle_module_task_name(module, gradle_task)
-            for gradle_task in ['assemble', 'test', 'lintRelease']
+            for gradle_task in ['assemble', 'assembleAndroidTest', 'test', 'lintRelease']
         ])
         task_definitions.append(
             BUILDER.craft_build_task(

--- a/components/feature/session-bundling/src/androidTest/java/mozilla/components/feature/session/bundling/OnDeviceSessionBundleStorageTest.kt
+++ b/components/feature/session-bundling/src/androidTest/java/mozilla/components/feature/session/bundling/OnDeviceSessionBundleStorageTest.kt
@@ -2,6 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+@file:Suppress("DEPRECATION")
+
 package mozilla.components.feature.session.bundling
 
 import android.content.Context

--- a/components/tooling/lint/build.gradle
+++ b/components/tooling/lint/build.gradle
@@ -29,3 +29,10 @@ task lint {
     // an empty lint task here.
   }
 }
+
+task assembleAndroidTest {
+  doLast {
+    // Do nothing. Like the `lint` task above this is just a dummy task so that this module
+    // behaves like our others and we do not need to special case it in automation.
+  }
+}

